task.sql

```diff
@@ -5,29 +5,50 @@ USE ShopDB;
 
 CREATE TABLE Countries (
     ID INT,
-    Name VARCHAR(50),
+    Name VARCHAR(30),
     PRIMARY KEY (ID)
 );
 
-CREATE TABLE ProductInventory (
-    ID INT,
-    ProductName VARCHAR(50),
-    WarehouseAmount INT,
-    WarehouseName VARCHAR(50),
-    WarehouseAddress VARCHAR(50), 
-    CountryID INT,
+CREATE TABLE Products (
+    ID INT AUTO_INCREMENT,
+    Name VARCHAR(30),
+    PRIMARY KEY(ID)
+);  
+
+CREATE TABLE Warehouses (
+  ID INT AUTO_INCREMENT,
+  Name VARCHAR(30),
+  Address VARCHAR(50),
+  CountryID INT,
 	FOREIGN KEY (CountryID) REFERENCES Countries(ID) ON DELETE NO ACTION,
     PRIMARY KEY (ID)
 );
 
+CREATE TABLE ProductInventory (
+    ID INT AUTO_INCREMENT,
+    ProductID INT,
+    FOREIGN KEY (ProductID) REFERENCES Products(ID) ON DELETE NO ACTION,
+    WarehouseAmount INT,
+    WarehouseID INT,
+    FOREIGN KEY (WarehouseID) REFERENCES Warehouses(ID) ON DELETE NO ACTION,
+    PRIMARY KEY (ID)
+);
 -- Populate test data
 
 INSERT INTO Countries (ID,Name)
 	VALUES (1, 'Country1');
 INSERT INTO Countries (ID,Name)
 	VALUES (2, 'Country2');
+
+INSERT INTO Products (Name)
+	VALUES ('AwersomeProduct');
+
+INSERT INTO Warehouses (Name, Address, CountryID)
+	VALUES ('Warehouse-1', 'City-1, Street-1', 1);
+INSERT INTO Warehouses (Name, Address, CountryID)
+	VALUES ('Warehouse-2', 'City-2, Street-2', 2);  
     
-INSERT INTO ProductInventory (ID,ProductName,WarehouseAmount,WarehouseName,WarehouseAddress,CountryID)
-	VALUES (1, 'AwersomeProduct', 2, 'Warehouse-1', 'City-1, Street-1',1);
-INSERT INTO ProductInventory (ID,ProductName,WarehouseAmount,WarehouseName,WarehouseAddress,CountryID)
-	VALUES (2, 'AwersomeProduct', 5, 'Warehouse-2', 'City-2, Street-2',2);
+INSERT INTO ProductInventory (ID,ProductID,WarehouseAmount,WarehouseID)
+	VALUES (1, 1, 1, 2);
+INSERT INTO ProductInventory (ID,ProductID,WarehouseAmount,WarehouseID)
+	VALUES (2, 1, 6, 2);
```

Comment:
![image](https://github.com/user-attachments/assets/4237f3c8-1f8a-4715-b56a-2fb735a61115)
Add Eof(End-of-file), need to add a blank line at the end